docs/src/cli-reference/TensileCreateLibrary.rst

```diff
@@ -84,6 +86,20 @@ Options
     Output master solution index in csv format including number 
     of kernels per architecture post build in csv format.
 
+Arguments
```

Comment:
Can we move these above the options? I feel like the required arguments should come first.